src/webviews/mongoClusters/documentView/documentView.tsx

```diff
@@ -203,6 +221,20 @@ export const DocumentView = (): JSX.Element => {
             .finally(() => {
                 setIsLoading(false);
             });
+
+        trpcClient.common.reportEvent
+            .mutate({
+                eventName: 'saveDocument',
+                properties: {
+                    ui: 'button',
+                },
+                measurements: {
+                    documentLength: editorContent.length,
+                },
+            })
+            .catch((error) => {
+                console.error('Failed to report event:', error);
```

Comment:
If this is error then either it should show message to user or send to telemetry. If this is debug info, please use console.debug or log.